recipes/trinity/2.1.1/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+BINARY=Trinity
+BINARY_HOME=$PREFIX/bin
+TRINITY_HOME=$PREFIX/opt/trinity-$PKG_VERSION
+
+cd $SRC_DIR
+
+make inchworm_target
+make chrysalis_target
+
+cd $SRC_DIR/trinity-plugins/
```

Comment:
The files in the 2.1.1 recipe are not actually the recipe I created, but the original recipe.